src/Router.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Introduce from './pages/Introduce/Introduce';
+import Product from './pages/Product/Product';
+import ProductList from './pages/ProductList/ProductList';
+
+const Router = () => {
+    return (
+        <BrowserRouter>
+                <Routes>                    
+                    <Route path="/" element={<Introduce/>} />
+                    <Route path="./pages/Product/Product" element={<Product/>} />
```

Comment:
```suggestion
                    <Route path="/Product" element={<Product/>} />
```

파일의 경로가 아닌 도메인의 경로를 설정해주셔야 합니다!